driver-service/src/main/java/com/modsen/driverservice/controllers/DriverController.java

```diff
@@ -0,0 +1,84 @@
+package com.modsen.driverservice.controllers;
+
+
+import com.modsen.driverservice.dto.*;
+import com.modsen.driverservice.services.DriverServiceImpl;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/v1/drivers")
+@RequiredArgsConstructor
+public class DriverController {
+
+
+    private final DriverServiceImpl driverService;
+
+
+    @GetMapping
+    public ResponseEntity<DriverResponseList> getAll(){
+        return ResponseEntity.ok(driverService.getAll());
+    }
+
+    @PostMapping
+    public ResponseEntity<DriverResponse> add(@RequestBody @Valid DriverRequest driverDto)
+    {
+        return ResponseEntity
+                .status(HttpStatus.CREATED)
+                .body(driverService.add(driverDto));
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity<DriverResponse> deleteById(@PathVariable(name = "id") Long id)
```

Comment:
Тоже самое, если название переменной и path variable совпадают, то можно явно не указывать в аннотации PathVariable:
```suggestion
   public ResponseEntity<DriverResponse> deleteById(@PathVariable Long id)
```